portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,116 @@
 <!DOCTYPE html>
 <html>
-  <head>
+<div style= "background-color: white;">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
+  
+    <header style="position: sticky;top: 0;">
+       
+    </header>
+    <main style = "font-size: 30px;" >
+        My portfolio: By Dalen Witthoft
+    </main>
+     
+</div>
+
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+    <img src = "/images/rimuru.png" style="width:5=100px;height:200px;">
+      <p>Click here to get a multilingual random message:</p>
+      <button onclick="addRandomGreeting()">Hello!</button>
       <div id="greeting-container"></div>
     </div>
+    <p style= "font-size: 20px">
+        Hi,
+        I am Dalen Witthoft and I am a rising sophmore at Boston University. I am majoring in 
+        Computer Science, but also have a passion for learning languages and about other cultures. I 
+        am currently studying Japanese and plan on expanding to Korean and Chinese before I graduate.
+        In my free time I love playing video games, watching anime, and watching K-Dramas. I also enjoy 
+        checking out what is going on in the competetive coding world as I did that throughout highschool. I
+        always find it interesting to see what strategies the top comptetive coders use to solve problems and 
+        see if I can adapt any of those approaches to my school work or personal projects. Thanks for checking out
+        my About Me website. :)
+    </p>
+    
+    <div>
+        <h1>Experience  </h1>
+        <ul><img src= "/images/check.png" style="width:10px;height:20px;"> 4 Years of CS club in HS </ul>
```

Comment:
thoughts on abstracting out the style="width:10px;height:20px;" to the css file under ul img {} to avoid repetition